### PR TITLE
Add Timezone Id to `getDefault` method.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

File: client/src/main/java/org/spine3/time/ZoneOffsets.java [hunk 1]
```diff
@@ -58,10 +59,12 @@ private ZoneOffsets() {
      * Obtains the ZoneOffset instance using timezone offset of the Java virtual machine.
      */
     public static ZoneOffset getDefault() {
-        final int seconds = TimeZone.getDefault()
+        final TimeZone timeZone = TimeZone.getDefault();
+        final int seconds = timeZone
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Please fix wrapping.

File: client/src/test/java/org/spine3/time/ZoneOffsetsShould.java [hunk 2]
```diff
@@ -46,8 +47,10 @@ public void has_private_constructor() {
 
     @Test
     public void create_default_instance_according_to_place() {
-        final int currentOffset = TimeZone.getDefault().getRawOffset()/(int)MILLIS_PER_SECOND;
+        final int currentOffset = timeZone.getRawOffset()/(int)MILLIS_PER_SECOND;
```
Comment (alexander-yevsyukov) [inline client/src/test/java/org/spine3/time/ZoneOffsetsShould.java]: We repeat this calculation over and over again. Why don't  create a utility method?

E.g. `getTimezoneZeconds()`.

File: client/src/main/java/org/spine3/time/ZoneOffsets.java [hunk 3]
```diff
@@ -56,19 +56,27 @@ private ZoneOffsets() {
...
+    public static ZoneOffset toZoneOffset(TimeZone timeZone) {
         return ZoneOffset.newBuilder()
-                         .setAmountSeconds(seconds)
+                         .setAmountSeconds(getOffsetInSeconds(timeZone))
                          .setId(nullToEmpty(timeZone.getID()))
                          .build();
     }
 
     /**
+     * Converts offset in millis to seconds.
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: `Obtains offset of the passed {@code TimeZone} in seconds.`

File: client/src/main/java/org/spine3/client/CommandFactory.java [hunk 4]
```diff
@@ -162,7 +165,7 @@ public CommandFactory build() {
             checkNotNull(actor, "`actor` must be defined");
 
             if (zoneOffset == null) {
-                setZoneOffset(ZoneOffsets.getDefault());
+                setZoneOffset(toZoneOffset(TimeZone.getDefault()));
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/client/CommandFactory.java]: Please have `getDefault()`.

Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/client/CommandFactory.java]: Fixed

File: client/src/main/java/org/spine3/time/ZoneOffsets.java [hunk 5]
```diff
@@ -69,14 +69,26 @@ public static ZoneOffset toZoneOffset(TimeZone timeZone) {
...
-     * Converts offset in millis to seconds.
+     * Obtains offset of the passed {@code TimeZone} in seconds.
      */
     public static int getOffsetInSeconds(TimeZone timeZone) {
         final int seconds = timeZone.getRawOffset() / (int) MILLIS_PER_SECOND;
         return seconds;
     }
 
     /**
+     * Obtains the ZoneOffset instance using timezone offset of the Java virtual machine.
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]:
```java
Obtains a {@code ZoneOffset} instance using default {@code TimeZone} of the Java virtual machine.

@see TimeZone#getDefault()
```

File: client/src/main/java/org/spine3/time/ZoneOffsets.java [hunk 6]
```diff
@@ -69,14 +69,26 @@ public static ZoneOffset toZoneOffset(TimeZone timeZone) {
...
+     */
+    public static ZoneOffset getDefault() {
+        final TimeZone timeZone = TimeZone.getDefault();
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Please have this:
```java
final TimeZone timeZone = TimeZone.getDefault();
return toZoneOffset(timeZone);
```

File: client/src/main/java/org/spine3/time/ZoneOffsets.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Fixed

File: client/src/test/java/org/spine3/time/ZoneOffsetsShould.java (hunk 2 above)
Comment (alexander-aleksandrov) [inline client/src/test/java/org/spine3/time/ZoneOffsetsShould.java]: Fixed

File: client/src/main/java/org/spine3/time/ZoneOffsets.java (hunk 3 above)
Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Fixed

File: client/src/main/java/org/spine3/time/ZoneOffsets.java (hunk 5 above)
Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Fixed

File: client/src/main/java/org/spine3/time/ZoneOffsets.java (hunk 6 above)
Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/time/ZoneOffsets.java]: Done

**Pull request merged**